src/interpreter.js

```
@@ -273,6 +277,19 @@ infixRules['||'] = infixRules['&&'] = (left, token, ctx) => {
   }
 };
 
+infixRules['in'] = (left, token, ctx) => {
+  let right = ctx.parse('in');
+  if (isObject(right) && !isArray(right)) {
+    right = Object.keys(right);
+  }
+
+  if (!(isArray(right) || isString(right))) {
+    throw expectationError('Infix: in', 'array/string/object to perform lookup');
+  }
+
+  return right.indexOf(left) !== -1;
```

As discussed in comment below... Should we throw an error if `left` if is not a string or number... Perhaps even only allow number if `right` is an array?

Or should we do deep equals for arrays and objects?